8.x/core-concepts/cruds.md

```diff
@@ -344,8 +328,6 @@ it should be made, which can include (breadcrumb, toolbar, fav icon, paginator,
                 </div>
             </div>
         </div>
-        
-        #Paginator
         @include('admin.templates.pagination', [
             "modelName" => "Todo",
             "lastPage" => $todos->lastPage(),
```

Comment:
Missing
"count" => $todos->perPage(),  
This command should also get changed.
40:19'